src/page/courier/TaskListPage.js

```diff
@@ -15,6 +16,29 @@ class TaskListPage extends Component {
     }
   }
 
+  componentDidMount() {
+    Settings.addListener('websocket:message', this.onWebSocketMessage.bind(this))
```

Comment:
You need to store `this.onWebSocketMessage.bind(this)` in a variable, or `removeListener` won't work as expected.